### ISSUE TITLE
Incorrect GraphQL data types

### ISSUE THREAD

**Issue by simonrabjohns** [verbb/formie#456]
**Description**

Some field properties are returning as String values when the field interface defines them as Boolean which results in introspection errors.

Might be related to fields that apply a default value for `includeInEmail` , `enableContentEncryption` and `enableConditions` that are responsible. Fields such as the hidden field that allow a value for `includeInEmail` to be set via the field settings lightswitch return with the correct data type. The SingleLineText does not seem to have this lightswitch in the field settings and returns a String instead of the expected Boolean.

**Steps to reproduce**
With a simple form containing a single line text field  and a hidden field in Graphiql run the query:

```query MyQuery {
  formieForm(handle: "contactForm") {
    id
    fields {
      handle
      ... on Field_SingleLineText {
        id
        includeInEmail
      }
    }
  }
}
```
and the response is 
```
{
  "data": {
    "formieForm": {
      "id": "4620",
      "fields": [
        {
          "handle": "fullName",
          "id": "303",
          "includeInEmail": "1"
        }
      ]
    }
  }
}
```
IncludeInEmail is returned as a String rather than a Boolean as specified in the Interface. 

Try the same query on the hidden field (which has the lightswitch setting to select a value for includeInEmail) and it returns a Boolean as expected:
```
query MyQuery {
  formieForm(handle: "contactForm") {
    id
    fields {
      handle
      ... on Field_Hidden {
        id
        name
        includeInEmail
      }
    }
  }
}
```
Result: 
```
{
  "data": {
    "formieForm": {
      "id": "4620",
      "fields": [
        {
          "handle": "fullName"
        },
        {
          "handle": "myHiddenField",
          "id": "304",
          "name": "my hidden field",
          "includeInEmail": true
        }
      ]
    }
  }
}
```
And querying for both fields together results in a conflicting data type error. 
```
{
  "errors": [
    {
      "message": "Fields \"includeInEmail\" conflict because they return conflicting types Boolean and String. Use different aliases on the fields to fetch both if this was intentional.",
      "extensions": {
        "category": "graphql"
      },
      "locations": [
        {
          "line": 7,
          "column": 9
        },
        {
          "line": 10,
          "column": 9
        }
      ]
    }
  ]
}
```

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: n/a
- Client-side Validation: n/a
- Custom Form Templates: n/a

**Additional info**
- Plugin version: 1.4.3
- Craft version: 3.6.12.1
- Multi-site: No

**Additional context**
Full list of fields with errors from schema introspection:
```
{
  "errors": [
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Address.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Address.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Agree.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Agree.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Agree.enableContentEncryption is type String.",
      "stringified": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Agree.enableContentEncryption is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Categories.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Categories.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Categories.enableContentEncryption is type String.",
      "stringified": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Categories.enableContentEncryption is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Checkboxes.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Checkboxes.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Checkboxes.enableContentEncryption is type String.",
      "stringified": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Checkboxes.enableContentEncryption is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Date.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Date.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Date.enableContentEncryption is type String.",
      "stringified": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Date.enableContentEncryption is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Dropdown.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Dropdown.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Dropdown.enableContentEncryption is type String.",
      "stringified": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Dropdown.enableContentEncryption is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Email.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Email.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Entries.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Entries.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Entries.enableContentEncryption is type String.",
      "stringified": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Entries.enableContentEncryption is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_FileUpload.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_FileUpload.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_FileUpload.enableContentEncryption is type String.",
      "stringified": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_FileUpload.enableContentEncryption is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Group.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Group.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Group.enableContentEncryption is type String.",
      "stringified": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Group.enableContentEncryption is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Heading.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Heading.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Heading.enableContentEncryption is type String.",
      "stringified": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Heading.enableContentEncryption is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.enableConditions expects type Boolean but craft_Field_Hidden.enableConditions is type String.",
      "stringified": "Interface field craft_FieldInterface.enableConditions expects type Boolean but craft_Field_Hidden.enableConditions is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Html.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Html.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Html.enableContentEncryption is type String.",
      "stringified": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Html.enableContentEncryption is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_MissingField.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_MissingField.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.enableConditions expects type Boolean but craft_Field_MissingField.enableConditions is type String.",
      "stringified": "Interface field craft_FieldInterface.enableConditions expects type Boolean but craft_Field_MissingField.enableConditions is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_MissingField.enableContentEncryption is type String.",
      "stringified": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_MissingField.enableContentEncryption is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_MultiLineText.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_MultiLineText.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Name.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Name.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Number.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Number.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Number.enableContentEncryption is type String.",
      "stringified": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Number.enableContentEncryption is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Phone.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Phone.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Radio.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Radio.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Radio.enableContentEncryption is type String.",
      "stringified": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Radio.enableContentEncryption is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Recipients.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Recipients.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Repeater.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Repeater.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Repeater.enableContentEncryption is type String.",
      "stringified": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Repeater.enableContentEncryption is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Section.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Section.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Section.enableContentEncryption is type String.",
      "stringified": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Section.enableContentEncryption is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_SingleLineText.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_SingleLineText.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Table.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Table.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Table.enableContentEncryption is type String.",
      "stringified": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Table.enableContentEncryption is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Tags.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Tags.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Tags.enableContentEncryption is type String.",
      "stringified": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Tags.enableContentEncryption is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Users.includeInEmail is type String.",
      "stringified": "Interface field craft_FieldInterface.includeInEmail expects type Boolean but craft_Field_Users.includeInEmail is type String."
    },
    {
      "message": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Users.enableContentEncryption is type String.",
      "stringified": "Interface field craft_FieldInterface.enableContentEncryption expects type Boolean but craft_Field_Users.enableContentEncryption is type String."
    }
  ]
}
```

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.3",
  "...": "..."
}
```

Then run `composer update`.

**Comment by simonrabjohns** [comment]
Great. thanks for the quick reply  - that's sorted it!

**Comment by engram-design** [comment]
Fixed in [1.4.4](https://github.com/verbb/formie/releases/tag/1.4.4)
